MapboxNavigation/InstructionPresenter.swift

```diff
@@ -23,7 +25,7 @@ class InstructionPresenter {
     
     func fittedAttributedComponents() -> [NSAttributedString] {
         guard let label = self.label else { return [] }
-        var attributedPairs = self.attributedPairs()
+        var attributedPairs = InstructionPresenter.attributedPairs(for: instruction, on: label, imageRepository: imageRepository, onImageDownload: completeShieldDownload)
```

Comment:
Let's work on this together tomorrow. I think there might be another way of introducing the seam we need, depending on the tests we need to write.